### DISCUSSION TITLE
After power off: Sleep Mode deactivated!

### DISCUSSION THREAD

**Discussion by DerBo19** [Blackymas/NSPanel_HA_Blueprint#980] [Q&A] [answered]
Sorry to bother you guys again!

After powering of my panel (it´s not on the wall at the moment), the sleep mode is deactivated. Also i can't find any option to set the sleep time neither any option to set the time when panel is jumping back to home page...

Can you please give me some advise! THANKS!

**Comment by edwardtfn** [comment, marked as answer]
Those options are under the device settings (Settings > Devices and Services > ESPHome).
![IMG_6377](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/94725493/c2dea981-28fc-4529-a89e-1bdda0e59b42)
